client/src/components/Workflow/Editor/NodeOutput.vue

```diff
@@ -1,6 +1,12 @@
 <template>
     <div class="form-row dataRow output-data-row">
-        <div v-if="showCallout" :class="['callout-terminal', output.name]" @click="onToggle">
+        <div
+            v-if="showCallout"
+            :class="['callout-terminal', output.name]"
+            @click="onToggle"
+            v-b-tooltip
+            title="Unchecked outputs will be hidden."
```

Comment:
```suggestion
            title="Unchecked outputs will be hidden and are not available as subworkflow outputs."
```
We really need to separate outputs from hidden. It's reasonable to have hidden workflow outputs.